### PR TITLE
redame content and some additional samples

### LINKED ISSUE #12726: Fill out all sections of Tables Readme
As referenced by Chris [here](https://github.com/Azure/azure-sdk-for-net/pull/12664#discussion_r438841202), the beginning sections of the main Tables readme should be filled out.

### PR DESCRIPTION
**Pull request by christothes**
closes #12726

### CODE REVIEW

File: sdk/tables/Azure.Data.Tables/readme.md [hunk 1]
```diff
@@ -1,34 +1,207 @@
...
+tableClient.DeleteEntity(partitionKey, rowKey);
+```
 
 ## Troubleshooting
 
-Content forthcoming
+When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP 
+status codes returned for [REST API][keyvault_rest] requests.
+
+For example, if you try to retrieve a create a table that already exists, a `409` error is returned, indicating "Conflict".
```
Comment (ellismg) [inline sdk/tables/Azure.Data.Tables/readme.md:176]: "if you try to retrieve a create a table that already exists" -> "if you try to create a table that already exists,"

File: sdk/tables/Azure.Data.Tables/readme.md [hunk 2]
```diff
@@ -1,34 +1,207 @@
...
+// Delete the entity given the partition and row key.
+
+tableClient.DeleteEntity(partitionKey, rowKey);
+```
 
 ## Troubleshooting
 
-Content forthcoming
+When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP 
+status codes returned for [REST API][keyvault_rest] requests.
```
Comment (ellismg) [inline sdk/tables/Azure.Data.Tables/readme.md:174]: This link target is not defined (looks like a copy/paste issue)

File: sdk/tables/Azure.Data.Tables/samples/Sample0Auth.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+var credential = new TableSharedKeyCredential(accountName, accountKey);
+
+var serviceClient = new TableServiceClient(
+    new Uri(storageUri),
+    credential);
+
+// Build a shared access signature with the Write and Delete permissions and access to all service resource types.
+
+TableAccountSasBuilder sasWriteDelete = serviceClient.GetSasBuilder(TableAccountSasPermissions.Write, TableAccountSasResourceTypes.All, new DateTime(2040, 1, 1, 1, 1, 0, DateTimeKind.Utc));
```
Comment (ellismg) [inline sdk/tables/Azure.Data.Tables/samples/Sample0Auth.md:92]: Should this be `TableAccountSasPermissions.Write | TableAccountSasPermissions.Delete` per the prose above?

**Review by ellismg — CHANGES_REQUESTED** [review]

File: sdk/tables/Azure.Data.Tables/samples/Sample0Auth.md (hunk 3 above)
Comment (christothes) [reply sdk/tables/Azure.Data.Tables/samples/Sample0Auth.md:92]: Write is inclusive of delete.

Comment (christothes) [reply sdk/tables/Azure.Data.Tables/samples/Sample0Auth.md:92]: However, I agree it may make the sample more clear.

**Review by ellismg — APPROVED** [review]
LGTM

**Pull request merged**